nni/retiarii/execution/base.py

```diff
@@ -8,27 +8,29 @@
 from typing import Any, Dict, Iterable, List
 
 from .interface import AbstractExecutionEngine, AbstractGraphListener
+from .utils import get_mutation_summary
 from .. import codegen, utils
 from ..graph import Model, ModelStatus, MetricData, Evaluator
 from ..integration_api import send_trial, receive_trial_parameters, get_advisor
 
 _logger = logging.getLogger(__name__)
 
-
 class BaseGraphData:
-    def __init__(self, model_script: str, evaluator: Evaluator) -> None:
+    def __init__(self, model_script: str, evaluator: Evaluator, mutation_summary: dict) -> None:
         self.model_script = model_script
         self.evaluator = evaluator
+        self.mutation_summary = mutation_summary
```

Comment:
This introduces a new term so it needs doc to describe what is "mutation summary".

Reply:
added